pkgs/by-name/uc/ucc/package.nix

```diff
@@ -33,43 +56,55 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   postPatch = ''
-
     for comp in $(find src/components -name Makefile.am); do
-      substituteInPlace $comp \
-        --replace "/bin/bash" "${stdenv.shell}"
+      if ! grep -q "/bin/bash" "$comp"; then
+        continue
+      fi
```

Comment:
Isn't this always false? I don't think /bin/bash ever exists in the sandbox (except when there's no sandbox)

Reply:
Some of the files don’t have `/bin/bash` in them so replacement would fail, so we skip them if they don’t match.

Reply:
Ahh, I just forgot grep's order of arguments